pkg/kubectl/cmd/testing/fake.go

```diff
@@ -481,6 +482,14 @@ func (f *FakeFactory) NewBuilder() *resource.Builder {
 	return resource.NewBuilder(mapper, f.CategoryExpander(), typer, resource.ClientMapperFunc(f.ClientForMapping), f.Decoder(true))
 }
 
+func (f *FakeFactory) NewUnstructuredBuilder() *resource.Builder {
```

Comment:
The code body is identical to `func (f *fakeAPIFactory) NewUnstructuredBuilder()`, other than the receiving type.

It also matches most of the code in `func (f *ring2Factory) NewUnstructuredBuilder()`

Is there any way to use the same code from one func?

Reply:
Reusing the code from `func (f *ring2Factory) NewUnstructuredBuilder()`: No. They are calling different functions underneath.

Reusing the code from `func (f *fakeAPIFactory) NewUnstructuredBuilder()`: That's possible. But it require refactoring the 2 separate implementations of `Factory` interface completely. I don't want to do it at least for now :/